requirements/ci.in

```diff
@@ -0,0 +1,6 @@
+# Requirements for running tests in Github Actions
+-c constraints.txt
+
+coveralls                 # Code coverage reporting
+tox                       # Virtualenv management for tests
+tox-battery               # Makes tox aware of requirements file changes
```

Comment:
do we really need `tox-battery`?

Reply:
I don't think so but it was being used with travis as well.